docs/get_started/integrating_libtropic/how_to_build/adding_to_project.md

```diff
@@ -24,14 +24,14 @@ add_subdirectory(${PATH_LIBTROPIC} "libtropic")
     target_link_libraries(tropic PUBLIC mbedtls)
     ```
 4. By default, libtropic does not link platform-specific code or its HAL, so it can be built as a static library. This is the consumer's responsibility:
-    1. Add HAL functions to sources and includes depending on your platform. For example, to include Linux SPI HAL to `MY_PROJECTS_SOURCES` source list:
+    1. For the chosen platform (e.g. Linux with HW SPI), add the correct subdirectory inside `libtropic/hal/`, which provides the corresponding HAL sources and include directories:
```

Comment:
"add the correct subdirectory" may sound like the user is supposed to create some directory